bigbone/src/main/kotlin/social/bigbone/api/method/Reports.kt

```diff
@@ -36,7 +36,7 @@ class Reports(private val client: MastodonClient) {
      * comment: A comment to associate with the report.
      */
     @Throws(BigboneRequestException::class)
-    fun postReport(accountId: Long, statusId: Long, comment: String): MastodonRequest<Report> {
```

Comment:
statusId should be `String` as well, right?

Reply:
Definitely! I also missed some other, more elusive uses of `Long` ids. :( I hope I now got all of them after doing a full text search on top of everything else. Please re-review.